agent-api/src/jvmMain/kotlin/JvmSerializable.kt

```diff
@@ -0,0 +1,3 @@
+package com.epam.drill.plugins.test2code.common.api
+
+actual typealias JvmSerializable = java.io.Serializable
```

Comment:
you use everywhere java.io.Serializable, but it has created typealias. So use typealias or remove it